apps/api-journeys/src/modules/video/video.ts

```diff
@@ -19,22 +21,40 @@ const typeDefs = gql`
     id: ID
     blockId: ID!
     state: VideoResponseStateEnum!
+    position: Float
+  }
+
+  interface Video {
```

Comment:
```suggestion
  interface VideoContent {
```

can we go with VIdeoContent instead of just video.